src/test/java/com/ericsson/eiffel/remrem/semantics/ServiceTest.java

```diff
@@ -157,12 +162,33 @@
     
     @InjectMocks
     EiffelTestExecutionRecipeCollectionCreatedEvent terEvent = new EiffelTestExecutionRecipeCollectionCreatedEvent();
+    
+    ManifestHandler manifestHandler;
+    Gav manifestGav = null;
       
     @Before
     public void setUp() throws Exception {
+    	manifestGav = new Gav();
+    	manifestHandler = mock(ManifestHandler.class);
         Attributes attributes = mock(Attributes.class);
         MockitoAnnotations.initMocks(this);        
-        when(attributes.getValue(anyString())).thenReturn("0.2.3");        
+        when(attributes.getValue(anyString())).thenReturn("0.2.3"); 
```

Comment:
postconstructor? BeforeClass possibilities for the below code, i.e one time read